lib/ansible/modules/system/crypttab.py

```diff
@@ -173,14 +170,12 @@ def __init__(self, path):
         if not os.path.exists(path):
             if not os.path.exists(os.path.dirname(path)):
                 os.makedirs(os.path.dirname(path))
-            open(path, 'a').close()
+        with open(path, 'a'):
```

Comment:
This needs to be indented, to make sure the file is only opened for appending when it doesn't yet exist.

Reply:
Great catch :face_with_head_bandage: Sorry. I'll change it. Weird why all tests pass with this, though.

Reply:
It doesn't destroy anything, but it touches the file unnecessarily. That's why the tests don't complain I guess.

Reply:
I updated it, however I'm actually not sure if the line of code does anything ...